src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -67,7 +67,9 @@ class MaskFormerFeatureExtractor(FeatureExtractionMixin, ImageFeatureExtractionM
             ImageNet std.
         ignore_index (`int`, *optional*, default to 255):
             Value of the index (label) to ignore.
-
+        num_labels (`int`, *optional*, defaults to 150):
+            The number of labels in the dataset. Needed to create the binary masks of shape `(batch, num_labels,
+            height, width)`.
```

Comment:
Maybe we can explain here why it defaults to 150.

Reply:
I think the default should be set to `None` and we should raise an error in the init if it's not been set: we can't have positional arguments the way the feature extractor works, but it's really something the user should set themselves. Note that whenever the user instantiates a feature extractor from a pretrained checkpoint, there won't be an error if the feature extractor config of that checkpoint sets `num_labels` (corresponding to its pretraining dataset).

Reply:
if we are going to raise an error, we can make it a required argument

Reply:
I was not aware of it, thanks for pointing it out!